src/main/java/eu/arrowhead/blacklist/BlacklistConstants.java

```diff
@@ -13,7 +13,6 @@ public final class BlacklistConstants {
 	public static final String VERSION_MANAGEMENT = "1.0.0";
```

Comment:
Can't comment there, but I think BLACKLIST.log is not something we want in the repository. Please delete it and add to ignore list